analyzer/src/funTest/assets/projects/synthetic/pipenv-expected-output.yml

```diff
@@ -95,25 +93,24 @@ packages:
 - id: "PyPI::itsdangerous:0.24"
   purl: "pkg:pypi/itsdangerous@0.24"
   authors:
-  - "Armin Ronacher"
+  - "Armin Ronacher <armin.ronacher@active-4.com>"
   declared_licenses:
-  - "BSD License"
+  - "UNKNOWN"
   declared_licenses_processed:
     unmapped:
-    - "BSD License"
-  description: "Various helpers to pass trusted data to untrusted environments and\
-    \ back."
+    - "UNKNOWN"
+  description: "It's Dangerous"
   homepage_url: "http://github.com/mitsuhiko/itsdangerous"
   binary_artifact:
-    url: "https://files.pythonhosted.org/packages/dc/b4/a60bcdba945c00f6d608d8975131ab3f25b22f2bcfe1dab221165194b2d4/itsdangerous-0.24.tar.gz"
+    url: ""
```

Comment:
Why's this missing?

Reply:
See the response on another similar comment.